src/test/java/com/zerobank/runners/CukesRunner.java

```diff
@@ -11,7 +11,7 @@
         features= "src/test/resources/features",
         glue = "com/zerobank/stepdefinitions",
         dryRun = false,
-        tags = "@wip"
+        tags = "@loginYA"
```

Comment:
normally you shouldnt commit cukesrunner class. cos this is common and everybody use their own tags